packages/layout/src/Legend.ts

```diff
@@ -227,6 +227,7 @@ export class Legend extends SVGWidget {
         this._legendOrdinal
             .orient(this.orientation())
             .title(this.title())
+            .labelWrap(this.labelMaxWidth())
```

Comment:
Will this handle the default `null` appropriately? Their documentation types it as `number`.

Reply:
Yes - I tested for that - 0 or null removes the wrapping